vignettes/dependencies.Rmd

```diff
@@ -28,6 +28,16 @@ inline <- function(x) {
 
 knitr::opts_chunk$set(
   collapse = TRUE)
+
+.here <- getwd()
+knitr::knit_hooks$set(inwd = function(before, options) {
```

Comment:
I think it would be worth moving a lot of this block into a common file that gets sourced by all the vignettes.

Reply:
yeah, good shout. I _think_ we can do that with `source()` - I'll give that a go and see...